Runtime/GLTF/GltFastGameObjectInstantiator.cs

```diff
@@ -41,8 +40,6 @@ public class GltFastGameObjectInstantiator : GameObjectInstantiator
             {
                 // Use Node GameObject for first Primitive
                 meshGo = m_Nodes[nodeIndex];
-                // Ready Player Me - Parent mesh to Avatar root game object
-                meshGo.transform.SetParent(m_Parent.transform);
```

Comment:
do you know, why was the parent changed to the root game object?

Reply:
ah good point, let me check. Probably I need to add this back in

Reply:
I added it back in. But when testing it out I couldn't notice any difference. Will check with robin once he is back